pyccl/halos/profiles/profile_base.py

```diff
@@ -232,15 +244,15 @@ def cumul2d(self, cosmo, r_t, M, a):
            \\Sigma(R')
```

Comment:
Is the LHS of the equation correctly displayed? In rtd, it is `Sigma(<R)`. Seems like there is something missing before the less than.

Reply:
no, that's right. That's a not-uncommon way of writing cumulative quantities.